docusaurus.config.js

```diff
@@ -230,18 +215,15 @@ module.exports = {
       "@docusaurus/preset-classic",
       {
         docs: {
-          sidebarPath: require.resolve("./sidebars.js"),
-          // Please change this to your repo.
-          editUrl:
-            "https://github.com/camunda/camunda-platform-docs/edit/main/",
+          lastVersion: "1.3",
+          includeCurrentVersion: false,
           beforeDefaultRemarkPlugins: [versionedLinks],
```

Comment:
In previous versions, we were removing this `beforeDefaultRemarkPlugins`. It wasn't needed, because there was no Optimize docs instance in those versions. 

Now we have both a main and optimize instance, so we need to preserve this line -- it's the thing that allows us to say `$docs$` or `$optimize$` in an anchor href and have the system figure out which version/URL to link to.